bootstrap.py

```diff
@@ -1053,12 +1092,74 @@ def exec_service(service, command, failonerror=True):
         migrate_systems(options.org, options.activationkey)
         if options.enablerepos:
             enable_repos()
+    elif options.new_capsule:
+        # > ELIF new_capsule and foreman_fqdn set, will migrate to other capsule
+        #
+        # > will replace CA certificate, reinstall katello-agent, gofer
+        # > will optionally update hostgroup and location
+        # > wil update system definition to point to new capsule for content,
+        # > Puppet, OpenSCAP and update Puppet configuration (if applicable)
+        # > MANUAL SIGNING OF CSR OR MANUALLY CREATING AUTO-SIGN RULE STILL REQUIRED!
+        # > API doesn't have a public endpoint for creating auto-sign entries yet!
+        if not is_registered():
+            print_error("This system doesn't seem to be registered to a Capsule at this moment.")
+            sys.exit(1)
+
+        # Make system ready for switch, gather required data
+        get_bootstrap_rpm(clean=True, unreg=False)
```

Comment:
I think we should call `install_prereqs()` before that. In 99.999% the cases it will be a no-op, but better safe than sorry.

Reply:
I'll slap it in there, but if sub-man is not installed, a system cannot be registered to a Satellite, be member of a hostgroup, have a content source, etc., to begin with, right? So aren't we catering for a non-existent use case?

Reply:
Yeah, it's quite a constructed case (I stumbled over it while trying to move a non-registered CentOS box before you added the "is registered" check).
The only case this can happen is: someone deliberately removed s-m *after* registering, thus mostly breaking the system.